packages/app-mobile/components/NoteBodyViewer/bundledJs/utils/addPluginAssets.ts

```diff
@@ -0,0 +1,76 @@
+import { RenderResultPluginAsset } from '@joplin/renderer/types';
+
+type PluginAssetRecord = {
+	element: HTMLElement;
+};
+const pluginAssetsAdded_: Record<string, PluginAssetRecord> = {};
+
+// Note that this function keeps track of what's been added so as not to
+// add the same CSS files multiple times.
+//
+// Shared with app-desktop/gui-note-viewer.
+//
+// TODO: If possible, refactor such that this function is not duplicated.
```

Comment:
To-do: Much of this is copied from `packages/app-desktop/gui/note-viewer/index.html`  -- it might make sense to refactor this. Doing so would, however require adding additional logic to build and include the `addPluginAssets` within `note-viewer/index.html`.